db/migrate/20190218180413_move_tag_value_to_tags.rb

```diff
@@ -0,0 +1,40 @@
+class MoveTagValueToTags < ActiveRecord::Migration[5.2]
+  def up
+    add_column :tags, :value, :string, :null => false, :default => ""
```

Comment:
we'll also need to change the unique index, now it's 
`["tenant_id", "namespace", "name"]`
we'll need
`["tenant_id", "namespace", "name", "value"]`

(you might need to drop&recreate it)

Reply:
:+1: